pkg/cli/zip_table_registry.go

```diff
@@ -378,18 +378,25 @@ var zipInternalTablesPerCluster = DebugZipTableRegistry{
 	"crdb_internal.system_jobs": {
 		// `payload` column may contain customer info, such as URI params
 		// containing access keys, encryption salts, etc.
-		nonSensitiveCols: NonSensitiveColumns{
+		customQueryUnredacted: `SELECT *, 
+			to_hex(payload) AS hex_payload, 
+			to_hex(progress) AS hex_progress 
+			FROM crdb_internal.system_jobs`,
+		customQueryRedacted: `SELECT 
 			"id",
 			"status",
 			"created",
+			'redacted' AS "payload",
```

Comment:
nit: one is '<redacted>' and the other is 'redacted'

Reply:
it wasn't but I also would rather not change it in the backport since, like you say, doesn't matter.